### DISCUSSION TITLE
Support for .bumrc file

### DISCUSSION THREAD

**Discussion by zorzysty** [owenizedd/bum#7] [Ideas]
Similarly to what nvm does with [.nvmrc](https://github.com/nvm-sh/nvm#nvmrc), bum could support installing a folder-specific bun version.

**Comment by owenizedd** [comment]
That's a good idea. I'll try to implement it

**Comment by owenizedd** [comment]
This feature deployed in v0.4.0

**Comment by TheMrKiko** [comment]
question: bun supports a file `.bun-version` for this purpose. why didn't you go with that file name? 
and suggestion: what if, when there is no .bumrc file in the directory, bum searched for a `.bun-version` file to use?

**Reply by zorzysty** [reply]
Just use [mise ](https://mise.jdx.dev/) mate
